data/test/scenarios/test_assert.cfg

```diff
@@ -27,3 +27,167 @@
         {RETURN ([not][true][/true][/not])}
     [/event]
 )}
+
+# Same as the above three but using the Lua unit_test module
+{GENERIC_UNIT_TEST "test_lua_assert" (
+    [event]
+        name = start
+        [lua]
+            code=<<
+                unit_test.assert(true, 'assert true')
+                unit_test.pass_if(true, 'return true')
+            >>
+        [/lua]
+    [/event]
+)}
+{GENERIC_UNIT_TEST "test_lua_assert_fail" (
+    [event]
+        name = start
+        [lua]
+            code=<<
+                unit_test.assert(false, 'assert false')
+                unit_test.pass_if(true, 'return true')
```

Comment:
What is this test testing? If it's testing `unit_test.assert`, then why is it using a `pass_if` instead of `pass`?

Reply:
It's a direct conversion of `test_assert` from the same file. I'm not sure why that test asserts and returns instead of asserting and succeeding.

Reply:
I feel the aim should be to test the Lua unit test framework, so it's more important to work out what it should test rather than to try and copy existing tests.

Reply:
Well, ultimately the choice of `pass` or `pass_if` here makes no real difference… so I guess I'll change it.